src/components/WorkTitle.js

```diff
@@ -103,26 +103,22 @@ class WorkTitle extends Component {
                                 </ul>
                             </h5>
                         </Col>
-                        <Col xs={6} sm={6}>
+                        <Col xs={12} sm={6}>
                             <RestaurateurCarousel images={carousel_kitchen}/>
                         </Col>
                     </Row></div>
                 <div className={'mt-5'}>
                     <h3>Restaurateur Dashboard</h3>
                     <Row className={'align-items-center'}>
-                        <Col xs={6} sm={6}>
+                        <Col xs={12} sm={6}>
                             <RestaurateurCarousel images={carousel_dashboard}/>
                         </Col>
                         <Col xs={12} sm={6}>
                             <h5 className='text-left'>
                                 <ul className="list-styled">
-                                    <li>Display your orders instantaneously as they arrive.
+                                    <li>Make your own menu this includes adding categories, individual items and prices.
```

Comment:
Manage your menu items with their prices and description

Add new categories or menus for special events like Ramadan.

Keep track of all the orders placed through our platform for back book review